targets/riot-stm32f4/Makefile.riot

```diff
@@ -29,24 +27,24 @@ EXT_CFLAGS += -Wno-error=format=
 all: libjerry riot-jerry
 
 libjerry:
-	mkdir -p $(INTERM)
-	mkdir -p $(OUTPUT)
+	mkdir -p $(BUILD_DIR)
 	mkdir -p $(COPYTARGET)
-	cmake -B$(INTERM) -H./ \
-	 -DEXTERNAL_PORT_DIR=UNDEFINED \
+	cmake -B$(BUILD_DIR) -H./ \
 	 -DENABLE_LTO=OFF \
-	 -DENABLE_VALGRIND=OFF \
-	 -DCMAKE_TOOLCHAIN_FILE=build/configs/toolchain_external.cmake \
+	 -DFEATURE_VALGRIND=OFF \
+	 -DCMAKE_TOOLCHAIN_FILE=cmake/toolchain_external.cmake \
+	 -DJERRY_LIBC=OFF\
+	 -DJERRY_CMDLINE=OFF\
 	 -DCOMPILER_DEFAULT_LIBC=ON \
+	 -DENABLE_ALL_IN_ONE=OFF\
 	 -DEXTERNAL_CMAKE_SYSTEM_PROCESSOR=armv7l-hf \
 	 -DEXTERNAL_CMAKE_C_COMPILER=arm-none-eabi-gcc \
 	 -DEXTERNAL_CMAKE_C_COMPILER_ID=GNU \
 	 -DEXTERNAL_COMPILE_FLAGS="$(EXT_CFLAGS)" \
-	 -DEXTERNAL_MEM_HEAP_SIZE_KB=$(JERRYHEAP)
+	 -DMEM_HEAP_SIZE_KB=$(JERRYHEAP)
 
-	make -C $(INTERM) $(TYPE).external
-	cp `cat $(INTERM)/$(TYPE).external/list` $(OUTPUT)/.
-	cp $(OUTPUT)/lib$(TYPE).jerry-core.a $(COPYTARGET)/libjerrycore.a
+	make -C$(BUILD_DIR) jerry-core
+	cp $(BUILD_DIR)/lib/libjerry-core.a $(COPYTARGET)/libjerrycore.a
```

Comment:
`$(BUILD_DIR) = build/riotstm32f4/` So after the cp command ..//lib/libjerry-core.a ...  is it okey?